services/ingestion/etl/inforegio/json/src/lib/transform.js

```diff
@@ -15,11 +15,16 @@ const getFundingArea = record =>
 
 // Formats date from DD/MM/YYYY to ISO 8601 date format.
 const formatDate = date => {
-  if (!date) return null;
+  if (!date || typeof date !== 'string') return null;
   const d = date.split(/\//);
-  if (d === null || d.length !== 3) return null;
-  if (d[2].length === 2) d[2] = `20${d[2]}`;
-  return new Date(d[2], d[1] - 1, d[0]).toISOString();
+  if (d.length !== 3) return null;
+  const [month, day, year] = d;
```

Comment:
If the stub is correct, then it should be `[day, month, year]`. Don't know which one is wrong, the stub or the transform?